platform/android/src/example_custom_layer.cpp

```diff
@@ -2,11 +2,114 @@
 #include <GLES2/gl2.h>
 
 #include <mbgl/util/logging.hpp>
-
+#include <mbgl/util/string.hpp>
```

Comment:
We should probably remove the `logging.hpp` and `string.hpp` dependencies. I don't think we should encourage custom layer implementations to depend on anything other than `custom_layer.hpp`.